snippets/lambda.js

```diff
@@ -1,5 +1,5 @@
 // https://stripe.com/docs/payments/without-card-authentication
-const stripe = require("stripe")("API_KEY")
+const stripe = require("stripe")(process.env.GATSBY_STRIPE_API_KEY) //"API_KEY"
```

Comment:
Note that this doesn't run in the context of Gatsby, so might be worthwhile just changing to `process.env.STRIPE_SECRET_KEY`

Reply:
Fixed in the next pull request, should I rename GATSBY_STORAGE_KEY to STORAGE_KEY also?